include/nbl/builtin/hlsl/workgroup/envmap.hlsl

```diff
@@ -0,0 +1,108 @@
+
+#ifndef _NBL_BUILTIN_HLSL_WORKGROUP_ENVMAP_IMPORTANCE_SAMPLING_INCLUDED_
+#define _NBL_BUILTIN_HLSL_WORKGROUP_ENVMAP_IMPORTANCE_SAMPLING_INCLUDED_
+
+namespace nbl
+{
+namespace hlsl
+{
+namespace workgroup
+{
+namespace envmap
```

Comment:
you're not in workgroup anything, also `envmap` should be `image_importance_sampling`